app/views/orders/show.html.erb

```diff
@@ -30,14 +31,23 @@
     </tr>
   <% @order.item_orders.each do |item_order|%>
     <tr>
+      <% if item_order.item.merchant_id == @merchant.id %>
     <section id = "item-<%=item_order.item_id%>">
-        <td><p><%=link_to item_order.item.name, "/items/#{item_order.item_id}"%></p></td>
-        <td><p><%= link_to item_order.item.merchant.name, "/merchants/#{item_order.item.merchant.id}"%></p></td>
-        <td><p><%= number_to_currency(item_order.price)%></p></td>
-        <td><p><%= item_order.quantity%></p></td>
-        <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
-      </section>
+        <% if item_order.status == 0 && item_order.quantity <= item_order.item.inventory %>
+          <td><p><%= link_to "Fulfill Order", "/item_orders/#{item_order.id}/fulfill"%></p></td>
+        <% elsif item_order.quantity >= item_order.item.inventory %>
+          <td><p>This order cannot be fulfilled due to lack of inventory.</p></td>
+        <% else %>
+          <td><p>Order has been fulfilled.</p></td>
+        <% end %>
+          <td><%= image_tag item_order.item.image %><p><%=link_to item_order.item.name, "/items/#{item_order.item_id}"%></p></td>
+          <td><p><%= link_to @merchant.name, "/merchants/#{@merchant.id}"%></p></td>
+          <td><p><%= number_to_currency(item_order.price)%></p></td>
+          <td><p><%= item_order.quantity%></p></td>
+          <td><p><%= number_to_currency(item_order.subtotal)%></p></td>
+        </section>
```

Comment:
Hey @lkriffell, 

We just talked about this, but I'll mark it here:
- It looks like you and I both relied heavily on the order's show page for both user and merchant.  Hopefully, we can fix this by adding a merchant's order show page?

Reply:
Changes have been made. You might want to look over it again just to make sure were not trying to use the same files in different ways